manifests/init.pp

```diff
@@ -81,6 +84,22 @@
   #Merged values provides by user and default values
   $_real_padoptions = merge($default_padoptions, $padoptions)
 
+  #Install choosing plugins
+  $plugins_list.each |$_pname, $_penable| {
+  if $_penable == true {
+    contain "::etherpad::plugins::${_pname}"
```

Comment:
mhm, why do you use contain + the class reference on the next line? Also please remove the trailing `::` in front of the class name.

Reply:
I tried to remove the `contain` and the code not compiled. That why I put it here.

Reply:
But I can remove `::`. Of course.

Reply:
@bastelfreak technically that's a leading `::`. Trailing is at the end. I think the `Class["etherpad::plugins::${_pname}"]` is not needed though.

Reply:
ups, yes. But you got my point :D